AssetRipper.TextureDecoder.TestGenerator/Program.cs

```diff
@@ -22,6 +22,7 @@ internal static class Program
 			new GenerationData(typeof(ColorRGB16), typeof(byte), 2),
 			new GenerationData(typeof(ColorRGB24), typeof(byte), 3),
 			new GenerationData(typeof(ColorRGB48), typeof(ushort), 6),
+			new GenerationData(typeof(ColorRGB32Half), typeof(Half), 4),
```

Comment:
This is causing a merge conflict. You need to rebase.

Reply:
You'll also need to add your new type to the [color source generation](https://github.com/AssetRipper/TextureDecoder/blob/07c653873b8508d0e3e62e40a37284f8e4ca6c11/AssetRipper.TextureDecoder.ColorGenerator/Program.cs#L85).